### PR TITLE
fix typings

### PR DESCRIPTION
**Pull request by na-2n**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-// * Contributed by Capuccino
+// * Contributed by Capuccino & noud02
 
-export module owo.js {
-    export interface Client {
-        key: 'key'
+export module "owo.js" {
+    export class Client {
+        public constructor (key: string);
+        public upload (path: string): Promise<any>;
```
Comment (aurieh) [inline index.d.ts]: any could be replaced by a type alias here (object keys are in whats-this/api)

File: index.d.ts [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
+        public shorten (url: string): Promise<any>;
```
Comment (aurieh) [inline index.d.ts]: same here

**Review by aurieh — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (na-2n) [reply index.d.ts]: will do, didnt have time to figure out the response so I used `any` there for now

Comment (deansheather) [reply index.d.ts]: https://documenter.getpostman.com/view/200155/owo-api/2QsY9k

**Comment by sr229** [comment]
LGTM, merging

**Pull request merged**